### PR TITLE
[Do Not Merge] Major Update (Fix Header, Footer, and Links)

### PR DESCRIPTION
**Pull request by thekodester**
Header and footer have been updated with correct links. Also I went with disabling French lanuage selection. More details to come.

### CODE REVIEW

**Comment by thekodester** [comment]
For transparence and it make it easier to review here a list of changes:

- Language select is disabled on most pages (this is due to the repo not following wet naming of pages with `-en` and `-fr`)
- Main Menu (now allows user to return to the main page)
- Breadcrumb (now allows user to return to documentation page)
- Prose and Footer links corrected
- WET styled 404 pages were added
- Gem files added for local testing
- Minor grammar issues fixed

**Comment by thekodester** [comment]
To run locally use the following terminal command bundle exec jekyll serve.

See https://help.github.com/en/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll for more details.

Word of warning when testing locally not all links well work properly and you may need to manually enter them (burden of local server testing).

**Comment by thekodester** [comment]
Updated jekyll tags:

```
---
published: true

// Determines the layout (can be en or fr)
layout: default-theme-wet-boew-en

// Changes the title page and the tab/bookmark name.
title: Demo
description: This is a demo.

// Disables the language selection.
languages: false

// Displays an error message that the page is written in English (used for untranslated French pages)
written_lang: en

// Displays the breadcrumb
hide_breadcrumb: false

// Date the file was last modified
date_modified: 2020-01-13

// Additional CSS or JS files
css_href: path/to/css/file.css
script_src: path/to/js/file1.js
script_src2: path/to/js/file1.js
script_src3: path/to/js/file1.js
---
```

**Comment by thekodester** [comment]
@duboisp @GormFrank @juleskuehn

File: assessment/2018-1.md [hunk 1]
```diff
@@ -3,7 +3,9 @@ published: true
 layout: default-theme-wet-boew-en
 title: Assessment of detail / summary support for AT
 description: Test results for NVDA 2018 and various browsers with recommendation
-modified: 2018-08-09
+languages: false
```
Comment (namjohn920) [inline assessment/2018-1.md]: could you clarify why 'languages' is set as false?

File: _includes/footer-theme-wet-boew-en.html [hunk 2]
```diff
@@ -1,59 +1,46 @@
...
+<h3>About</h3>
+<ul class="list-unstyled">
+<li><a href="../../wet-boew/index-en.html#about">About the Web Experience Toolkit</a></li>
+<li><a rel="external" href="https://www.canada.ca/en/treasury-board-secretariat/topics/government-communications.html">About the Web Standards</a></li>
+</ul>
+</section>
+<section class="col-sm-3">
+<h3>News</h3>
+<ul class="list-unstyled">
+<li><a href="https://github.com/wet-boew/wet-boew-documentation/pulse">Recent project activity</a></li>
```
Comment (namjohn920) [inline _includes/footer-theme-wet-boew-en.html]: href has to be https://github.com/wet-boew/wet-boew/pulse .

File: _includes/footer-theme-wet-boew-en.html [hunk 3]
```diff
@@ -1,59 +1,46 @@
...
+<li><a href="https://github.com/wet-boew/wet-boew-documentation/graphs">Project statistics</a></li>
```
Comment (namjohn920) [inline _includes/footer-theme-wet-boew-en.html]: href has to be https://github.com/wet-boew/wet-boew/graphs .
Same thing should be done for french version.

File: _includes/footer-theme-wet-boew-en.html [hunk 4]
```diff
@@ -1,59 +1,46 @@
...
-
-<![endif]-->
-{% if page.script_pgbttm_src and page.script_pgbttm_src contains '.js' %}
-<script src='{{ page.script_pgbttm_src }}'></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
+<script src="https://wet-boew.github.io/wet-boew/wet-boew/js/wet-boew.min.js"></script>
+{% if page.script_src and page.script_src contains '.js' %}
+<script src='{{ page.script_src }}'></script>
 {% endif %}
-{% if page.script_pgbttm2_src and page.script_pgbttm2_src contains '.js' %}
```
Comment (namjohn920) [inline _includes/footer-theme-wet-boew-en.html]: I assume you changed the name for sake of having more readability right?

Comment (thekodester) [reply _includes/footer-theme-wet-boew-en.html]: Yes

**Comment by thekodester** [comment]
Due to recent PRs I'm having issues with the conflicts when trying to make the requested changes. Should I just make a new PR?

**Comment by namjohn920** [comment]
> Due to recent PRs I'm having issues with the conflicts when trying to make the requested changes. Should I just make a new PR?

Yes please do so. Thanks for your work!

**Pull request closed without merging**